File: client/src/main/java/at/oenb/dltrouting/domain/RoutingEntry.java
package at.oenb.dltrouting.domain;

import lombok.*;

import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder(toBuilder = true)
public class RoutingEntry {

	private String id;
	private BankOperation bankOperation;
	private Service service;
	private BIC counterPartyBic;
	private OffsetDateTime validFrom;
	private OffsetDateTime validTo;
	private BIC cooperationBic;

}

File: webapp/code/src/app/models/routingEntry.ts
import { BankOperation } from './bankOperation';
import { Services } from './services';
import { BIC } from './bic';


export class RoutingEntry {
  id: string;
  bankOperation: string;
  service: string;
  counterPartyBic: BIC;
  validFrom: string;
  validTo: string;
  cooperationBic: BIC;
}

export class RoutingEntryExtended {
  id: string;
  bankOperation: string;
  service: string;
  counterPartyBic: BIC;
  validFrom: string;
  validTo: string;
  cooperationBic: BIC;
  bankname: string;
}

